lib/core/utils/parse-crossorigin-stylesheet.js

```diff
@@ -19,6 +19,7 @@ axe.utils.parseCrossOriginStylesheet = function parseCrossOriginStylesheet(
 ) {
 	const axiosOptions = {
 		method: 'get',
+		timeout: axe.constants.preload.timeout,
```

Comment:
Defaulting to `preload.timeout` constant means something passed as a configuration will be ignored.

Eg:
```
axe.run({
	runOnly: {
		...
	},
	preload: {
		assets: ['cssom', ...],
		timeout: 10000
	}
})
```

In this case I specifically want timeout to be after x seconds and not a fallback constant.

> Note: This is still not right, but I will open a PR for this.